### PR TITLE
Get only first cluster for watch CRD MultiCassKop

### PR DESCRIPTION
**Pull request by erdrix**
Fast fix, operator stop to watch remote CRD MultiCasskop. 
I try to find a more explicit declaration of masters and remotes, using struct or something else.

### CODE REVIEW

File: multi-casskop/pkg/controller/multi-casskop/multi-casskop_controller.go [hunk 1]
```diff
@@ -41,6 +41,7 @@ import (
 type Clusters struct {
 	Name    string
 	Cluster *cluster.Cluster
+	IsMaster bool
```
Comment (erdrix) [inline multi-casskop/pkg/controller/multi-casskop/multi-casskop_controller.go]: Update Clusters struct to add master and remote information.

**Comment by erdrix** [comment]
I fixed the bug with remotes watch, into the commit : https://github.com/Orange-OpenSource/cassandra-k8s-operator/pull/156/commits/ebfbbbe6579dcf648de76e40faae402161c202a7, feature is ready to be reviewed.

File: multi-casskop/pkg/controller/multi-casskop/models/clusters.go [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+import (
+	"admiralty.io/multicluster-controller/pkg/cluster"
+	"fmt"
+	apicmc "github.com/Orange-OpenSource/cassandra-k8s-operator/multi-casskop/pkg/apis"
+	apicc "github.com/Orange-OpenSource/cassandra-k8s-operator/pkg/apis"
+	"github.com/sirupsen/logrus"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Cluster define k8s clsuter information we need.
```
Comment (cscetbon) [inline multi-casskop/pkg/controller/multi-casskop/models/clusters.go]: typo

File: multi-casskop/pkg/controller/multi-casskop/models/clusters.go [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+type Cluster struct {
+	Name string
+	Cluster *cluster.Cluster
+}
+
+// Clusters defined each kubernetes cluster we want to connect on it.
```
Comment (cscetbon) [inline multi-casskop/pkg/controller/multi-casskop/models/clusters.go]: defined -> defines

**Comment by cscetbon** [comment]
@erdrix a few minor comments and it's all good

**Pull request merged**